olive/package_config.py

```diff
@@ -0,0 +1,36 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
the file location seems be different with the PR https://github.com/microsoft/Olive/pull/1051. which PR is the final one?

Reply:
#1051 is meant to be a follow up after this PR merges. I still have to rebase that PR.